synfig-studio/src/gui/docks/dock_timetrack2.cpp

```diff
@@ -233,6 +233,31 @@ void Dock_Timetrack2::setup_tool_palette()
 		action_button_map[tool_button->get_name()] = tool_button;
 		tool_item_group->add(*tool_button);
 	}
+
+	struct interpolationButtonInfo {
+		std::string icon_name;
+		std::string name;
+		synfig::Interpolation interpolation;
+	};
+
+	const std::vector<interpolationButtonInfo> interp_buttons_info{
+		{"interpolation_type_clamped_icon", N_("Clamped"), synfig::INTERPOLATION_CLAMPED},
+		{"interpolation_type_tcb_icon", N_("TCB"), synfig::INTERPOLATION_TCB},
+		{"interpolation_type_const_icon", N_("Constant"), synfig::INTERPOLATION_CONSTANT},
+		{"interpolation_type_ease_icon", N_("Ease In/Out"), synfig::INTERPOLATION_HALT},
+		{"interpolation_type_linear_icon", N_("Linear"), synfig::INTERPOLATION_LINEAR}
+	};
+
+	for (const auto & interp_button_info: interp_buttons_info) {
+		Gtk::Image* image= Gtk::manage(new Gtk::Image());
+		image->set_from_icon_name(interp_button_info.icon_name, Gtk::IconSize::from_name("synfig-small_icon_16x16"));
+		Gtk::ToolButton *tool_button = manage(new Gtk::ToolButton(*image, interp_button_info.name));

```

Comment:
`_(interp_button_info.name)` to actually translate it :)

Reply:
oops for some reason I thought N_("") was enough :"